systemtest/aggregation_test.go

```diff
@@ -77,10 +71,6 @@ func TestTransactionAggregation(t *testing.T) {
 		espoll.TermQuery{Field: "processor.event", Value: "transaction"},
 	)
 
-	// Make sure apm-server.aggregation.txmetrics metrics are published. Metric values are unit tested.
-	doc := getBeatsMonitoringStats(t, srv, nil)
-	assert.True(t, gjson.GetBytes(doc.RawSource, "beats_stats.metrics.apm-server.aggregation.txmetrics").Exists())
```

Comment:
We probably discussed this in the past and I forgot. We're not relying on this in any part of stack monitoring, supportability dashboards, or product telemetry, right?

Reply:
There is one internal dashboard tracking overflows. We will need apm-aggregation to emit metrics around overflows, before we create a new dashboard for it.

Reply:
Will do in https://github.com/elastic/apm-aggregation/issues/75